garage/experiment/experiment_wrapper.py

```diff
@@ -66,7 +66,7 @@ def run_experiment(argv):
     parser.add_argument(
         '--snapshot_mode',
         type=str,
-        default='all',
+        default='last',
```

Comment:
please keep `'all'` as default -- i think this is the behavior most people expect.

Reply:
Fixed.